src/werkzeug/test.py

```diff
@@ -1288,6 +1292,10 @@ class TestResponse(Response):
         Test client methods always return instances of this class.
     """
 
+    default_mimetype = None
+    """Don't assume a mimetype, instead use whatever the response provides
```

Comment:
Use a normal `#` comment here, don't override the docstring.

Reply:
Changed.